src/app/calculator/system-and-equipment-curve/regression-equations/regression-equations.component.html

```diff
@@ -25,9 +25,23 @@
                     <span [innerHTML]="baselineEquipmentRegressionEquation"></span>
                 </td>
             </tr>
-            <tr *ngIf="selectedEquipmentCurveFormView == 'Data'">
+            <tr *ngIf="selectedEquipmentCurveFormView == 'Data' && baselineEquipmentCurveByDataRSquared > 0">
                 <td class="pl-2">r<sup>2</sup>: {{baselineEquipmentCurveByDataRSquared | number: '.1-4'}} </td>
             </tr>
+            <tr *ngIf="selectedEquipmentCurveFormView == 'Equation'">
+                <td class="pl-2"><span class="text-muted small">R²: N/A</span></td>
+            </tr>
+            <tr *ngIf="baselineEquipmentPowerRegressionEquation">
+                <td class="pl-2">
+                    <span [innerHTML]="baselineEquipmentPowerRegressionEquation"></span>
+                </td>
+            </tr>
+            <tr *ngIf="selectedEquipmentCurveFormView == 'Data' && baselineEquipmentPowerCurveByDataRSquared > 0">
+                <td class="pl-2">r<sup>2</sup>: {{baselineEquipmentPowerCurveByDataRSquared | number: '.1-4'}} </td>
+            </tr>
+            <tr *ngIf="selectedEquipmentCurveFormView == 'Equation' && baselineEquipmentPowerRegressionEquation">
+                <td class="pl-2"><span class="text-muted small">R²: N/A</span></td>
```

Comment:
If there's no reason to switch cases (r --> R) from a scientific/engineering perspective, use lowercase "r" for consistency across the markup. Also prefer superscript tags for consistency instead of the unicode